devops/roles/postgresql-restore/defaults/main.yml

```diff
@@ -0,0 +1,15 @@
+postgresql_restore_dir: /tmp/postgresql-restore
+postgresql_port: 5432
+postgresql_cluster_version: 9.5
```

Comment:
Are we overriding the postgresql_cluster_version for druid? What version of postgresql are we running in Druid?

Reply:
We are running postgresql version 9.5 and hence this is ok.